src/components/GroupAll/groupFilter/RangeSlidrFilter.tsx

```diff
@@ -0,0 +1,189 @@
+  }, [minValue, maxValue]);
+
+  return (
+    <RangeContainer>
+      <RangeSlider>
+        <RangeSelected
+          left={`${((minValue - min) / (max - min)) * 100}%`}
+          right={`${100 - ((maxValue - min) / (max - min)) * 100}%`}
+        />
+        <ThumbValueContainer>
+          <ThumbValueLeft left={`${((minValue - min) / (max - min)) * 100}%`}>
+            {minValue}명
+          </ThumbValueLeft>
+          <ThumbValueRigth
+            right={`${100 - ((maxValue - min) / (max - min)) * 100}%`}
+          >
+            {maxValue}명
+          </ThumbValueRigth>
+        </ThumbValueContainer>
+      </RangeSlider>
+      <RangeInputContainer>
+        <RangeInput
+          type="range"
+          className="min"
+          min={min}
+          max={max}
+          value={minValue}
+          step={step}
+          onChange={(e) => handleInputChange(e, 'min')}
+        />
+        <RangeInput
+          type="range"
+          className="max"
+          min={min}
+          max={max}
+          value={maxValue}
+          step={step}
+          onChange={(e) => handleInputChange(e, 'max')}
+        />
+      </RangeInputContainer>
+    </RangeContainer>
+  );
+};
+
+export default RangeSliderFilter;
+
+const RangeContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 55rem;
+`;
+
+const RangeSlider = styled.div`
+  position: relative;
+
+  height: 0.7rem;
+
+  border-radius: 9.9rem;
+  background-color: ${({ theme }) => theme.colors.gray400};
+`;
+
+const RangeSelected = styled.span<{ left: string; right: string }>`
+  position: absolute;
+  right: ${({ right }) => right};
+  left: ${({ left }) => left};
+
+  height: 100%;
+
+  border-radius: 9.9rem;
+  background-color: ${({ theme }) => theme.colors.codrive_purple};
+`;
+
+const RangeInputContainer = styled.div`
+  position: relative;
+`;
+
+const RangeInput = styled.input`
+  position: absolute;
+  top: -7px;
+
+  width: 100%;
+  height: 0.5rem;
+  margin: 0 auto;
+
+  background: none;
+  pointer-events: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+
+  &::-webkit-slider-thumb {
+    position: relative;
+
+    width: 1.9rem;
+    height: 1.9rem;
+
+    border: 0.4rem solid ${({ theme }) => theme.colors.white};
+    border-radius: 50%;
+    background-color: ${({ theme }) => theme.colors.codrive_purple};
+    pointer-events: auto;
+    -webkit-appearance: none;
+  }
+
+  &::-moz-range-thumb {
+    position: relative;
+
+    width: 1.9rem;
+    height: 1.9rem;
+
+    border: 0.4rem solid ${({ theme }) => theme.colors.white};
+    border-radius: 50%;
+    background-color: ${({ theme }) => theme.colors.codrive_purple};
+    pointer-events: auto;
+  }
+`;
+
+const ThumbValueContainer = styled.div`
+  color: ${({ theme }) => theme.colors.white};
+`;
+
+const ThumbValueLeft = styled.div<{ left?: string; right?: string }>`
+  position: absolute;
+  top: 2.5rem;
+
+  width: 2.5rem;
+
+  ${({ left }) => left && `left: ${left};`}
```

Comment:
ASK: 오 이거 아래처럼 안써도 잘 동작하나요??
```suggestion
  ${({ left }) => left && css`left: ${left};`}
```

Reply:
넵 잘 동작은 하는 거 같습니다!